.github/workflows/lumen.yml

```diff
@@ -0,0 +1,30 @@
+name: plugin-big-commerce-di
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  laravel-tests:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Copy .env
+      run: php -r "file_exists('.env') || copy('.env.example', '.env');"
+    - name: Install Dependencies
+      run: composer install -q --no-ansi --no-interaction --no-scripts --no-progress --prefer-dist
+    - name: Generate key
+      run: php artisan key:generate
+    - name: Create Database
+      run: |
+        mkdir -p database
+        touch database/database.sqlite
+    - name: Execute tests (Unit and Feature tests) via PHPUnit
+      env:
+        DB_CONNECTION: sqlite
+        DB_DATABASE: database/database.sqlite
+      run: vendor/bin/phpunit
```

Comment:
After creating the sqlite database, don't you need to run the migrations as well?

Reply:
![Screen Shot 2020-11-09 at 14 22 38](https://user-images.githubusercontent.com/511559/98546399-0c7f2f80-2297-11eb-8dec-581b64668017.png)